nilearn/datasets/description/spm_multimodal.rst

```diff
@@ -10,32 +10,32 @@ See :func:`nilearn.datasets.fetch_spm_multimodal_fmri`.
 Notes
 -----
 The example shows the analysis of an :term:`SPM` dataset studying face perception.
-The analysis is performed in native space.
-Realignment parameters are provided with the input images,
-but those have not been resampled to a common space.
+The images are in native space: they have not been resampled to a common space.
 
 The experimental paradigm is simple, with two conditions:
 viewing a face image or a scrambled face image,
 supposedly with the same low-level statistical properties,
 to find face-specific responses.
 
+Images were acquired with a repetition time of 2 seconds.
+
```

Comment:
Mention download link and also that the dataset also exists on openneuro.

https://openneuro.org/datasets/ds000117

Also there is fmriprep derivatives of the dataset on openneuro: https://github.com/OpenNeuroDerivatives/ds000117-fmriprep